diff -up mailcap-2.1.31/mime.types.webm mailcap-2.1.31/mime.types
--- mailcap-2.1.31/mime.types.webm	2009-11-17 22:13:33.000000000 +0100
+++ mailcap-2.1.31/mime.types	2011-06-03 12:22:39.374226992 +0200
@@ -1286,6 +1286,7 @@ text/cache-manifest				manifest
 text/x-pod					pod
 text/x-setext					etx
 text/x-vcard					vcf
+video/webm					webm
 video/x-flv					flv
 video/x-ms-asf					asx
 video/x-ms-wm					wm
